packages/cardano-services/src/migrationDataSource.ts

```diff
@@ -7,7 +7,7 @@ export { DataSource } from 'typeorm';
 
 export const AppDataSource = createDataSource({
   connectionConfig: {
-    database: 'projection',
+    database: 'stake_pool',
```

Comment:
We'll be using this for other projections as well, I suggest rolling this change back, or pass db name as cli arg

Reply:
Pass in as argument, since we need control over the name

Reply:
This data source configuration is used only for development

Reply:
Won't there be a conflict if the dev env has multiple services setup?

Reply:
I wasn't able to pass the database name as command line argument since this file is only _required_: `typeorm-ts-node-commonjs` consumes the command line arguments and exits with error if it find some of them not recognized.
Anyway I found an alternative solution using environment variables.
45fdfde88 should solve this and improve the procedure to generate new migration and the relative documentation